app/models/chargeback/report_options.rb

```diff
@@ -62,14 +62,14 @@ def report_time_range
       ts = Time.now.in_time_zone(tz)
       case interval
       when 'daily'
-        start_time = (ts - start_interval_offset.days).beginning_of_day.utc
```

Comment:
We might want to put a comment above `ts = Time.now.in_time_zone(tz)` saying something like `# Use local time for determining the start and end of the date range since we use and are charged for consumption in local time`.  Or something that clearly states why we're using local time here.